lighthouse-core/gather/fetcher.js

```diff
@@ -168,10 +166,16 @@ class Fetcher {
       timeoutHandle = setTimeout(() => reject(new Error(errorMessage)), timeout);
     });
 
-    return Promise.race([
+    const racePromise = Promise.race([
       timeoutPromise,
       requestInterceptionPromise,
     ]).finally(() => clearTimeout(timeoutHandle));
+
+    this.driver.evaluateAsync(`${injectIframe}(${JSON.stringify(url)})`, {
+      useIsolation: true,
+    }).catch(err => log.error('Fetcher', err));
+
+    return racePromise;
```

Comment:
```suggestion
    const injectionPromise = this.driver.evaluateAsync(`${injectIframe}(${JSON.stringify(url)})`, {
      useIsolation: true,
    });

    const [fetchResult] = await Promise.all([racePromise, injectionPromise]);
    return fetchResult;
```

Reply:
this is what I meant.

Reply:
That's where I stopped at because I thought tossing out the rest of the `Promise.all` return value was so unusual I must have misunderstood. sgtm :)

Reply:
fyi (nbd)

![image](https://user-images.githubusercontent.com/4071474/86188278-06ac5d00-baf3-11ea-9640-01fe43f18cba.png)

I expected a string here.